app/src/main/java/com/eventyay/organizer/ui/views/DatePicker.java

```diff
@@ -29,10 +37,23 @@ public void setValue(String value) {
             observableValue.set(value);
             String format = DateUtils.FORMAT_DATE_COMPLETE;
 
+            DateFormat dateFormat = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss", Locale.getDefault());
+            Date date = null;
+            try {
+                date = dateFormat.parse(value);
+            } catch (ParseException e) {
+                e.printStackTrace();
+            }
+
+            Calendar calendar = Calendar.getInstance();
+            calendar.setTime(date);
+            int hour = calendar.get(Calendar.HOUR_OF_DAY);
+            int minute = calendar.get(Calendar.MINUTE);
```

Comment:
Why are these steps needed?

Reply:
To collect hours and minutes, could have directly applied as parameters... Or substring can be used as the format is same, but this would benefit if format is changed in future